### ISSUE TITLE
No ground stations are created when running with Kopernikus without the Real Solar System mod

### ISSUE THREAD

**Issue by anon12341234** [DRVeyl/RealAntennas#31]
It looks like issue #20 isn't fixed; I'm running the following:

* KSP 1.8.1
* RealAntennas 1.2
* [Kopernicus 1.8.1-1](https://github.com/Kopernicus/Kopernicus/tree/release-1.8.1-1)
* ModuleManager 4.1.3

and I'm not getting any ground stations, and I can't transfer any data when I'm on the launch pad. I tried messing around with the config files, but no matter what I try I can't get any ground stations to appear. I tried adding the Real Solar System mod, and with that mod the ground stations **do** appear.

(And before you ask - my intention is not to run Kopernicus without a planet pack - I want to use the Outer Planets Mod, but that doesn't affect this issue.)

**Comment by anon12341234** [comment]
Okay, I figured it out. Creating a new mod with the following `.cfg` file fixes the issue (probably not a proper fix):

```
@Kopernicus:AFTER[Kopernicus]
{
    @Body[Kerbin]
    {
        @PQS
        {
            Mods
            {
            }
        }
    }
}
```

So it looks like Real Antennas is doing an `@Mods`, and it doesn't work as the `Mods` key doesn't exist as this is how the skeleton is defined in Kopernicus:

```
    Body
    {
        name = Kerbin
        identifier = Squad/Kerbin
        Orbit
        {
            referenceBody = Squad/Sun
        }
        Template
        {
            name = Kerbin
            removeProgressTree = false
        }
        Properties
        {
            ScienceValues
            {

            }
        }
        ScaledVersion
        {

        }
        Atmosphere
        {

        }
        PQS
        {

        }
    }
```

With the Real Solar System mod it works because that mod replaces all of the planets wholesale, and includes the `Mods` key under `PQS`, so Real Antennas' `@Mods` works there.

**Comment by DRVeyl** [comment]
Yep, that sounds correct.  Thanks.  I will edit the patch in stock.cfg to create the Mods node if it does not exist, and wrap that into the next release.

**Comment by DRVeyl** [comment]
Fixed with https://github.com/DRVeyl/RealAntennas/commit/649fd21c4001f709708c85b778babc7c34934362 and released in 1.2.1
